src/libraries/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -399,10 +399,8 @@ public bool Equals(Complex value)
 
         public string ToString(IFormatProvider? provider) => ToString(null, provider);
 
-        public string ToString([StringSyntax(StringSyntaxAttribute.NumericFormat)] string? format, IFormatProvider? provider)
-        {
-            return string.Format(provider, "<{0}; {1}>", m_real.ToString(format, provider), m_imaginary.ToString(format, provider));
-        }
+        public string ToString([StringSyntax(StringSyntaxAttribute.NumericFormat)] string? format, IFormatProvider? provider) =>
+            $"<{m_real.ToString(format, provider)}; {m_imaginary.ToString(format, provider)}>";
```

Comment:
>; [](http://example.com/codeflow?start=50&length=1)

Is it intentional to change the `,` to `;`?

Reply:
Which comma did I change?  If it's part of the output, certainly wasn't intentional, but I'm also not seeing it.

Reply:
never mind. my bad reading the whole old format :-)